qt/aqt/qt/compat.py

```diff
@@ -0,0 +1,1265 @@
+QStyle.PM_LayoutRightMargin = QStyle.PixelMetric.PM_LayoutRightMargin
+QStyle.PM_LayoutTopMargin = QStyle.PixelMetric.PM_LayoutTopMargin
+QStyle.PM_LayoutVerticalSpacing = QStyle.PixelMetric.PM_LayoutVerticalSpacing
+QStyle.PM_SliderLength = QStyle.PixelMetric.PM_SliderLength
+QStyle.PM_SliderSpaceAvailable = QStyle.PixelMetric.PM_SliderSpaceAvailable
+QStyle.SP_ArrowDown = QStyle.StandardPixmap.SP_ArrowDown
+QStyle.SP_ArrowUp = QStyle.StandardPixmap.SP_ArrowUp
+QStyle.SP_BrowserReload = QStyle.StandardPixmap.SP_BrowserReload
+QStyle.SP_DialogApplyButton = QStyle.StandardPixmap.SP_DialogApplyButton
+QStyle.SP_DialogCancelButton = QStyle.StandardPixmap.SP_DialogCancelButton
+QStyle.SP_DialogOpenButton = QStyle.StandardPixmap.SP_DialogOpenButton
+QStyle.SP_DirClosedIcon = QStyle.StandardPixmap.SP_DirClosedIcon
+QStyle.SP_DirLinkIcon = QStyle.StandardPixmap.SP_DirLinkIcon
+QStyle.SP_DirOpenIcon = QStyle.StandardPixmap.SP_DirOpenIcon
+QStyle.SP_FileDialogContentsView = QStyle.StandardPixmap.SP_FileDialogContentsView
+QStyle.SP_FileIcon = QStyle.StandardPixmap.SP_FileIcon
+QStyle.SP_TrashIcon = QStyle.StandardPixmap.SP_TrashIcon
+QStyle.State_Children = QStyle.StateFlag.State_Children
+QStyle.State_Selected = QStyle.StateFlag.State_Selected
+QStyle.SC_SliderGroove = QStyle.SubControl.SC_SliderGroove
+QStyle.SC_SliderHandle = QStyle.SubControl.SC_SliderHandle
+QStyle.SE_ItemViewItemText = QStyle.SubElement.SE_ItemViewItemText
+QSystemTrayIcon.DoubleClick = QSystemTrayIcon.ActivationReason.DoubleClick
+QSystemTrayIcon.Trigger = QSystemTrayIcon.ActivationReason.Trigger
+QSystemTrayIcon.Critical = QSystemTrayIcon.MessageIcon.Critical
+QSystemTrayIcon.Information = QSystemTrayIcon.MessageIcon.Information
+QSystemTrayIcon.NoIcon = QSystemTrayIcon.MessageIcon.NoIcon
+QSystemTrayIcon.Warning = QSystemTrayIcon.MessageIcon.Warning
+QTabBar.RightSide = QTabBar.ButtonPosition.RightSide
+QTabWidget.North = QTabWidget.TabPosition.North
+QTabWidget.South = QTabWidget.TabPosition.South
+QTabWidget.West = QTabWidget.TabPosition.West
+QTabWidget.Rounded = QTabWidget.TabShape.Rounded
+QTextEdit.NoWrap = QTextEdit.LineWrapMode.NoWrap
+QToolButton.DelayedPopup = QToolButton.ToolButtonPopupMode.DelayedPopup
+QToolButton.InstantPopup = QToolButton.ToolButtonPopupMode.InstantPopup
+QWizard.NoBackButtonOnLastPage = QWizard.WizardOption.NoBackButtonOnLastPage
+QWizard.NoCancelButton = QWizard.WizardOption.NoCancelButton
+QWizard.ClassicStyle = QWizard.WizardStyle.ClassicStyle
+QWizard.ModernStyle = QWizard.WizardStyle.ModernStyle
+
+
+# Miscellaneous
+
+QDBus.AutoDetect = QDBus.CallMode.AutoDetect
+QHostAddress.LocalHost = QHostAddress.SpecialAddress.LocalHost
+QPrinter.Millimeter = QPrinter.Unit.Millimeter
+
+# Globally alias removed PyQt5.Qt to PyQt6.QtCore.Qt
+##########################################################################
+
+from . import qt5qt
+
+sys.modules["PyQt5.Qt"] = qt5qt
+sys.modules["PyQt6.Qt"] = qt5qt
```

Comment:
Could you explain what you're trying to accomplish here? If I've read it right, it will attempt to import PyQt5 after PyQt6 has loaded, which will cause crashes if both are available in the Python environment (at least on Macs)